spec/features/stories_spec.rb

```diff
@@ -47,6 +47,31 @@
 
   end
 
+  describe "story history" do
+
+    let!(:story) do
```

Comment:
Since you're not using the `story` variable, you could put it inside a `before`.